.gitignore

```diff
@@ -28,3 +28,4 @@ release-docs.zip
 /tmp/checksum.txt
 api_compat_report/
 logs/
+**/src/main/assets/sdk_versions/
```

Comment:
Let's mention this in PR description

Reply:
@ank27 also I guess we need to explicitly mention in PR description and also in our `DEVELOPING.md` file that JDK is _required_ now.

Reply:
why do we need to mention `sdk_versions` are ignored, these are used internally by plugin and not for public-facing changes?

Reply:
yeah, agree about `sdk_versions`